src/Files/Views/ColumnShellPage.xaml

```diff
@@ -123,6 +123,11 @@
             Key="F1"
             Invoked="KeyboardAccelerator_Invoked"
             IsEnabled="{x:Bind IsCurrentInstance, Mode=OneWay}" />
+        <KeyboardAccelerator
+            Key="K"
+            Invoked="KeyboardAccelerator_Invoked"
+            IsEnabled="{x:Bind IsCurrentInstance, Mode=OneWay}"
+            Modifiers="Control" />
```

Comment:
Can you change this to `ctr` + `shft` + `k`?

Reply:
will do

Reply:
pr addressing this here https://github.com/files-community/Files/pull/7901